azure-cs-botservice/README.md

```diff
@@ -0,0 +1,81 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Azure Bot Service with Application Insights
+
+Starting point for building Azure Bot Service hosted in Azure App Service.
+
+Provisions Azure Bot Service, Azure Bot Channel registration and Azure Application Insights to be used in combination
+with App Service - registering Azure AD Microsoft Application with secret.  This will deploy the echo bot within the ~/bot/bot.zip file - you can replace the contents with your own bot.  Simply publish the bot to a local folder, zipping the contents and replacing this file.
```

Comment:
It might be nicer to commit the source code for the bot directly to the example folder and build it as part of the readme. This way, people can start changing the bot more easily.

Reply:
Added bot as source code folder and updated readme.md